### ISSUE TITLE
Linux Release

### ISSUE THREAD

**Issue by realdep** [Exodus-20-2/Lucid#9]
Possible?

**Comment by Exodus-20-2** [comment]
@realdep Yes it is and im planning on adding it in the next update along with a step-by-step guide on how to use lucid on linux.

**Comment by MouseHubMC** [comment]
stfu just use wine it worked fine for me
